app/config/routes.js

```diff
@@ -3,11 +3,14 @@ var ReactRouter = require('react-router');
 var Router = ReactRouter.Router;
 var Route = ReactRouter.Route;
 var hashHistory = ReactRouter.hashHistory;
+var IndexRoute = ReactRouter.IndexRoute;
 var Main = require('../components/Main');
+var TypeTextContainer = require('../containers/TypeTextContainer');
 
 var routes = (
   <Router history={hashHistory}>
     <Route path="/" component={Main}>
+		<IndexRoute component={TypeTextContainer} />
```

Comment:
nit: code style thing, indentation is split between 2 spaces and 1 tab it looks like